src/zarr/storage/memory.py

```diff
@@ -14,9 +14,25 @@
     from zarr.core.common import AccessModeLiteral
 
 
-# TODO: this store could easily be extended to wrap any MutableMapping store from v2
```

Comment:
Should this TODO stay?

Reply:
In a recent conversation, we decided not to do this. So this removal is reflecting that decision.